pkg/daemon/ceph/osd/kms/volumes_test.go

```diff
@@ -77,3 +77,43 @@ func TestTLSSecretVolumeAndMount(t *testing.T) {
 		})
 	}
 }
+func TestVaultTokenVolumeAndMount(t *testing.T) {
```

Comment:
Tests should be named `TestFunction` where "Function" is the name of the function under test.

```suggestion
func TestVaultTokenVolume(t *testing.T) {
```